theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -430,7 +421,8 @@ def test_log1pexp_to_softplus(self):
 
         out = T.log(1 + T.exp(x))
         f = theano.function([x], out, mode=self.m)
-        assert hasattr(f.maker.fgraph.outputs[0].tag, 'trace')
+
+        assert check_stack_trace(f, ops_to_check='all')
         topo = f.maker.fgraph.toposort()
         assert len(topo) == 1
         assert isinstance(topo[0].op.scalar_op,
```

Comment:
Yes, can you open a ticket for that? Then just refer to that ticket in a comment here, instead of actually calling check_stack_trace.